package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "browser-sync start --server --directory",
-    "sass": "node-sass src/scss/main.scss src/css/main.css",
+    "sass": "node-sass src/scss/listing--rating-panel.scss src/css/listing--rating-panel.css",
```

Comment:
Same thing here. This clobbers _everyone's_ config (and I know you're not the only one to do it). If you'd like to [send _additional_ flags to the `npm start`](https://docs.npmjs.com/cli/run-script) script, you can do so like this:

    $> npm start -- --startPath src/trending.html
    $> npm run sass -- --watch="src/scss/listing--rating-panel.scss"